packages/quill/src/crypto/auth.sagas.ts

```diff
@@ -67,6 +76,49 @@ export function* createPasswordWorker({ payload }: PayloadAction<string>) {
   yield put(push(ROUTE_PATHS.SETUP_ACCOUNT));
 }
 
+export function* getAccountPrivateKey(account: TUuid) {
```

Comment:
Does this need to be exported?

Reply:
I intended to test it, but forgot. So yes